### ISSUE TITLE
Unable to load the API config when using the alertad.conf with volume mount

### ISSUE THREAD

**Issue by novicejava1** [alerta/docker-alerta#311]
**Issue Summary**
Unable to load the API config when the configuration setting alertad.conf is loaded using volume mount

**Environment**
- OS: Fedora 32
- API version: 8.5.0
- Deployment: Docker
- Database: Postgres

- Server config:
  Auth enabled? Yes
  Auth provider? Basic
  Customer views? No
  (or provide the `/config` endpoint output)

- web UI version: 8.5.0
- CLI version: NA

**To Reproduce**
I am using the following custom web ui docker image with the below docker-compose.yml file

```
$ cat docker-compose.yml_working 
version: '2.1'
services:
  web:
    #build: .
    image: novicejava1/alerta-customweb1
    volumes:
      - $PWD/server.crt:/etc/ssl/certs/server.crt
      - $PWD/server.key:/etc/ssl/certs/server.key
    ports:
      - "8080:8080"
    depends_on:
      - db
    environment:
      - DEBUG=1  # remove this line to turn DEBUG off
      - DATABASE_URL=postgres://postgres:postgres@db:5432/monitoring
      - AUTH_REQUIRED=True
      - ADMIN_USERS=admin@alerta.io,devops@alerta.io #default password: alerta
      - ADMIN_KEY=demo-key
      - PLUGINS=reject,blackout,normalise,enhance
    restart: always
  db:
    image: postgres
    volumes:
      - ./pg-data:/var/lib/postgresql/data
    environment:
      POSTGRES_DB: monitoring
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
    restart: always
```

I am able to launch the web ui and api with the following docker compose file. But if i try to put the environment variable within alertad.conf file and load it using a volume as shown in the below docker-compose file i am unable to launch config endpoint throwing 500 internal server exception. Details below.

```
$ cat docker-compose.yml_notworking 
version: '2.1'
services:
  web:
    #build: .
    image: novicejava1/alerta-customweb2
    volumes:
      - $PWD/server.crt:/etc/ssl/certs/server.crt
      - $PWD/server.key:/etc/ssl/certs/server.key
      - $PWD/config/alertad.conf:/app/alertad.conf
    ports:
      - "8080:8080"
    depends_on:
      - db
    #environment:
    #- DEBUG=1  # remove this line to turn DEBUG off
    # - DATABASE_URL=postgres://postgres:postgres@db:5432/monitoring
    # - AUTH_REQUIRED=True
    # - ADMIN_USERS=admin@alerta.io,devops@alerta.io #default password: alerta
    # - ADMIN_KEY=demo-key
    # - PLUGINS=reject,blackout,normalise,enhance
    restart: always
  db:
    image: postgres
    volumes:
      - ./pg-data:/var/lib/postgresql/data
    environment:
      POSTGRES_DB: monitoring
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
    restart: always
```

```
alertad.conf -
$ cat config/alertad.conf_backup 
# General Settings
DEBUG = True
#BASE_URL = '/risk/'

# Auth Provider Settings
SECRET = "^Bpa%i8_nCAc8fI4l9)nhn2EG2!@GJga"
AUTH_REQUIRED = True
ADMIN_USERS=admin@alerta.io,devops@alerta.io #default password: alerta
ADMIN_PASSWORD='admin@2021'
ADMIN_KEY='demo-key'

# Severity Settings
SEVERITY_MAP = {
    'fatal': 0,
    'critical': 1,
    'major': 2,
    'minor': 3,
    'warning': 4,
    'indeterminate': 5,
    'cleared': 5,
    'normal': 5,
    'ok': 5,
    'informational': 6,
    'debug': 7,
    'trace': 8,
    'unknown': 9
}
DEFAULT_NORMAL_SEVERITY = 'normal'  # 'normal', 'ok', 'cleared'
DEFAULT_PREVIOUS_SEVERITY = 'indeterminate'

# Database Settings
DATABASE_URL = 'postgres://postgres:postgres@db:5432/monitoring'

# Plugin Settings
PLUGINS = ['reject', 'blackout', 'geoip', 'normalise', 'msteams']
```

For web app issues, include any web browser JavaScript console errors. - Errors from the browser console

```
Browser Console error -
xhr.js:177 GET https://localhost:8080/api/config 500 (Internal Server Error)
(anonymous) @ xhr.js:177
t.exports @ xhr.js:13
t.exports @ dispatchRequest.js:52
Promise.then (async)
u.request @ Axios.js:61
r.forEach.u.<computed> @ Axios.js:76
(anonymous) @ bind.js:9
value @ config.ts:67
(anonymous) @ config.ts:28
Promise.then (async)
value @ config.ts:26
cd49 @ main.ts:32
c @ bootstrap:89
0 @ app.81929175.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.81929175.js:1
config.ts:34 Error: Request failed with status code 500
    at t.exports (createError.js:16)
    at t.exports (settle.js:17)
    at XMLHttpRequest.f.onreadystatechange (xhr.js:62)
createError.js:16 Uncaught (in promise) Error: Request failed with status code 500
    at t.exports (createError.js:16)
    at t.exports (settle.js:17)
    at XMLHttpRequest.f.onreadystatechange (xhr.js:62)
```

Browser Network trace - config.json
Request URL: https://localhost:8080/risk/config.json
Request Method: GET
Status Code: 200 OK

Browser Network trace - confg
Request URL: https://localhost:8080/api/config
Request Method: GET
Status Code: 500 Internal Server Error
Remote Address: [::1]:8080
Referrer Policy: strict-origin-when-cross-origin
Connection: keep-alive
Content-Type: text/plain
Date: Tue, 22 Jun 2021 07:41:37 GMT
Server: nginx/1.20.1
Transfer-Encoding: chunked
**Expected behavior**

**Screenshots**
![browser_console_error](https://user-images.githubusercontent.com/28642296/122887616-b4e19000-d35e-11eb-931f-c6d1ac1e0e71.png)
![browser_network_error](https://user-images.githubusercontent.com/28642296/122887631-b8751700-d35e-11eb-9a62-dffb71c34fc3.png)

**Additional context**
Custom Nginx server config for custom subpath

```
        server {
                listen 8080 default_server ssl;
		ssl_certificate /etc/ssl/certs/server.crt;
		ssl_certificate_key /etc/ssl/certs/server.key;
		ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
    		ssl_ciphers         HIGH:!aNULL:!MD5;


                access_log /dev/stdout main;
                access_log /dev/stdout scripts;

                location ~ /api {
                        include /etc/nginx/uwsgi_params;
                        uwsgi_pass unix:/tmp/uwsgi.sock;

                        proxy_set_header Host $host:$server_port;
                        proxy_set_header X-Real-IP $remote_addr;
                        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                }

                root /web;
                index index.html;
                location /risk/ {
			alias /web/; 
                        try_files $uri $uri/ /index.html;
                }
        }
```

NOTE: Please provide as much information about your issue as possible.
Failure to provide basic details about your specific environment make
it impossible to know if an issue has already been fixed, can delay a
response and may result in your issue being closed without a resolution.

**Comment by satterly** [comment]
Please provide the HTTP error response from the API or the Docker log file that shows the 500 error. Thanks.

**Comment by satterly** [comment]
Fixed by #339
